WebService/v1/Models/DeviceModelApiModel/DeviceModelApiModel.cs

```diff
@@ -191,6 +196,47 @@ public void ValidateInputRequest(ILogger log)
             this.Simulation.ValidateInputRequest(log);
         }
 
+        public List<string> ValidationHelper()
+        {
+            List<string> errors = new List<string>();
+
+            // We accept empty string, null and 'Custom' values
```

Comment:
Did you mean 'We don't accept null, empty string and 'Custom' values'?

Reply:
We accept `Type`:
 - string empty
 - null
 - 'Custom'